hoomd/hpmc/UpdaterBoxMC.cc

```diff
@@ -284,7 +284,7 @@ inline bool UpdaterBoxMC::box_resize_trial(Scalar Lx,
 
     const Scalar kT = (*m_mc->getKT())(timestep);
 
-    if (allowed && p < exp(-(delta_beta_H + delta_U_pair + delta_U_external) / kT) + log_V_term)
```

Comment:
Are you sure this change is correct? The cassandra docs show $P\\Delta V$ (which HOOMD calls `delta_beta_H`) multiplied by $\beta$. You can check in Frenkel and Smit to verify.

If you found errors in testing, double check the sign on the `log_V_term`.